backend/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "start": "nodemon --exec \"ts-node\" app.ts",
```

Comment:
`nodemon` is not listed as a dependency

Reply:
Just to clarify: I don't think we should expect any tooling to be installed globally, so we should add everything we use as a dependency. This makes things simpler for deployment, for example.